notebook/nbextensions.py

```diff
@@ -299,118 +355,300 @@ def _config_file_name_default(self):
     def install_extensions(self):
         if len(self.extra_args)>1:
```

Comment:
this is triggering an error when I tried to install using --py in a sys-prefix... I mean:
```
$ jupyter nbextension install --py nb_conda --sys-prefix /home/damian/miniconda3/envs/notebook_dev
[InstallNBExtensionApp] Unrecognized JSON config file version, assuming version 1
Traceback (most recent call last):
  File "/home/damian/miniconda3/envs/notebook_dev/bin/jupyter-nbextension", line 9, in <module>
    load_entry_point('notebook', 'console_scripts', 'jupyter-nbextension')()
  File "/home/damian/miniconda3/envs/notebook_dev/lib/python3.5/site-packages/jupyter_core/application.py", line 267, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/home/damian/miniconda3/envs/notebook_dev/lib/python3.5/site-packages/traitlets/config/application.py", line 589, in launch_instance
    app.start()
  File "/media/data/devel/jupyter_project/notebook/notebook/nbextensions.py", line 525, in start
    super(NBExtensionApp, self).start()
  File "/home/damian/miniconda3/envs/notebook_dev/lib/python3.5/site-packages/jupyter_core/application.py", line 256, in start
    self.subapp.start()
  File "/media/data/devel/jupyter_project/notebook/notebook/nbextensions.py", line 384, in start
    self.install_extensions()
  File "/media/data/devel/jupyter_project/notebook/notebook/nbextensions.py", line 357, in install_extensions
    raise ValueError("only one nbextension allowed at a time.  Call multiple times to install multiple extensions.")
ValueError: only one nbextension allowed at a time.  Call multiple times to install multiple extensions.
```